lib/pxcontext.js

```diff
@@ -105,6 +105,7 @@ class PxContext {
     }
 
     areCredentialsCompromised() {
+        // eslint-disable-next-line eqeqeq
```

Comment:
Why not just:
`return (this.pxde && this.pxde['breached_account'] && this.pxdeVerified)`

instead of the != null and then you won't need the // eslint-disable-next-line eqeqeq

Not blocking or something but just wanted to note